app/models/register.rb

```diff
@@ -79,6 +79,19 @@ def show_category_link?
     register_phase == 'Beta' && category.present?
   end
 
+  def is_register_published_by_dcms?
+    Authority.where(id: authority_id).first&.government_organisation_key === 'D5'
```

Comment:
I think we can express this more simply as: 
```
authority.government_organisation_key == 'D5'
```

Reply:
I think it might work if you use safe navigation operator e.g.
```
authority&.government_organisation_key == 'D5'
```

Reply:
Cheers!